.github/workflows/MacOS.yml

```diff
@@ -0,0 +1,53 @@
+name: MacOS
+on:
+  push:
+    branches:
+    - develop
+    paths-ignore:
+    - README.md
+  pull_request:
+    branches:
+    - develop
+    paths-ignore:
+    - README.md
+
+jobs:
+  MacOS:
+    runs-on: macos-latest
+    env:
+      FC: gfortran-11
+
+    steps:
+    
+    - name: install-deps
+      run: |
+        brew update
+        brew install doxygen
+        pip3 install setuptools
+        pip3 install numpy
+        pip3 install netCDF4
+        sudo ln -sf /usr/local/bin/gfortran-11 /usr/local/bin/gfortran
+          
+    - name: checkout-bufr
+      uses: actions/checkout@v2
+      with: 
+        path: bufr
+
+    - name: build-bufr
+      run: |
+        cd bufr
+        mkdir build 
+        cd build
+        cmake ..
+        make -j2 VERBOSE=1
```

Comment:
lines 41 and 42 are now redundant and should be removed

Reply:
Good catch.